form/src/main/java/org/dhis2/form/ui/event/UiEventFactoryImpl.kt

```diff
@@ -44,7 +30,7 @@ class UiEventFactoryImpl(
         var uiEvent: RecyclerViewUiEvents? = null
         try {
             uiEvent = when (uiEventType) {
-                DATE_TIME -> {
+                /*DATE_TIME -> {
```

Comment:
This commented code is no longer needed? or was just for development purposes?

Reply:
Not used at all :)